modules/table/src/test/java/org/apache/ignite/internal/table/distributed/schema/FullTableSchemaTest.java

```diff
@@ -29,17 +29,8 @@
 import org.junit.jupiter.api.Test;
 
 class FullTableSchemaTest {
-    @Test
-    void sameSchemasHaveEmptyDiff() {
-        CatalogTableColumnDescriptor column = someColumn("a");
-
-        var schema1 = new FullTableSchema(1, 1, List.of(column));
-        var schema2 = new FullTableSchema(2, 1, List.of(column));
-
-        TableDefinitionDiff diff = schema2.diffFrom(schema1);
-
-        assertThat(diff.isEmpty(), is(true));
-    }
+    private static final String TABLE_NAME1 = "test1";
```

Comment:
Why have you deleted the test?

Reply:
Because the tested method is not needed and I was going to delete it as well, but forgot to do so. Thanks for noticing, I've removed the tested method.